src/Buckle/Buckle/CodeAnalysis/Binding/BoundExpressionTypes.cs

```diff
@@ -91,6 +94,8 @@ private BoundBinaryOperator(SyntaxType type, BoundBinaryOperatorType opType, Bou
             BoundTypeClause.Int, BoundTypeClause.Bool),
         new BoundBinaryOperator(SyntaxType.GREATER_THAN_EQUALS_TOKEN, BoundBinaryOperatorType.GreatOrEqual,
             BoundTypeClause.Int, BoundTypeClause.Bool),
+        new BoundBinaryOperator(SyntaxType.GREATER_THAN_GREATER_THAN_GREATER_THAN_EQUALS_TOKEN, BoundBinaryOperatorType.GreatOrEqual,
+            BoundTypeClause.Int, BoundTypeClause.Bool),
```

Comment:
Should not be the GreatorOrEqual BoundBinaryOperatorType, and should not have the BoundTypeClause.Bool parameter at the end.

However this doesn't matter because remember compound assignment operators are all handled by this point so this code will never be reached. These 2 lines should be removed.